### DISCUSSION TITLE
游戏崩溃，求求大佬了

### DISCUSSION THREAD

**Discussion by 56356erw** [Meloong-Git/PCL#2399] [提问] [answered]
[错误报告-2023-8-1_12.46.13.zip](https://github.com/Hex-Dragon/PCL2/files/12224732/-2023-8-1_12.46.13.zip)

**Comment by SALTWOOD** [comment]
安装了mod没？

**Comment by hejiehao** [comment]
重装游戏

**Comment by 56356erw** [comment]
一个旅行地图mod，然后点那个地图设置就崩了。然后就进不去了，谢谢回复
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2023年08月01日 12:54 |
| 收件人 | Hex-Dragon/PCL2 ***@***.***> |
| 抄送人 | 56356erw ***@***.***>,
Author ***@***.***> |
| 主题 | Re: [Hex-Dragon/PCL2] 游戏崩溃，求求大佬了 (Discussion #2399) |

安装了mod没？

**Reply by SALTWOOD** [reply, marked as answer]
试试删除config文件夹下的带journeymap的文件/文件夹

**Comment by Griseo-bh3rd** [comment]
--height, 480, --width, 854,在启动参数里出现了两次，删掉一个试试？

**Reply by Super-XiaoNian** [reply]
没毛病，我猜他一定是用PCL和HMCL轮流启动了一遍

**Reply by Griseo-bh3rd** [reply]
这么久没回复，要不关了？

**Comment by hejiehao** [comment]
重装游戏试试
